documentation/source/building.rst

```diff
@@ -290,6 +290,11 @@ The following variables are makefile variables, not environment variables.
       A list of the Verilog source files to include.
       Paths can be absolute or relative; if relative, they are interpreted as relative to the Makefile's location.
 
+.. make:var:: VERILOG_INCLUDE_DIRS
+
+      A list of the Verilog directories to search for include files.
+      Paths can be absolute or relative; if relative, they are interpreted as relative to the Makefile's location.
```

Comment:
Technically it's relative to where `make` is called. i.e. `make -f path/to/sim/Makefile` will not include `path/to/sim` in the name.

Reply:
Feel free to edit it to your specifications

Reply:
@ktbarrett 

BTW the path information is identical to the one given for `VERILOG_SOURCES` which is 2 lines above this, so maybe that should be edited too

Reply:
I made a follow on issue to correct them all.